libr/core/cmd_anal.c

```diff
@@ -7449,6 +7459,9 @@ static bool cmd_anal_refs(RCore *core, const char *input) {
 				r_cons_println (pj_string (pj));
 				pj_free (pj);
 			}
+			else if (input[1] == '?') { // axt?
```

Comment:
Did you actually tried if this works? Doesn't `axt?` doesn't print anything if reference list is not empty.

Reply:
I had tried calling it from before the check for the list, just after the `case 't':` That caused:
```
Usage: axt[?gq*]  find data/code references to this address
| axtj [addr]  find data/code references to this address and print in json format
| axtg [addr]  display commands to generate graphs according to the xrefs
| axtq [addr]  find and list the data/code references in quiet mode
| axt* [addr]  same as axt, but prints as r2 commands
| axt?         Show this help
entry0 0x8048337 [DATA] push main
```
if the seek was on `main`, so I was trying out another way.

Well, now, since you asked me again, I tried adding a `break;` statement.
```
	case 't': { // "axt"
		if (input[1] == '?') { // axt?
			r_core_cmd_help (core, help_msg_axt);
			break;
		}	
```

Which apparently works?
```
[0x00001060]> axt?
Usage: axt[?gq*]  find data/code references to this address
| axtj [addr]  find data/code references to this address and print in json format
| axtg [addr]  display commands to generate graphs according to the xrefs
| axtq [addr]  find and list the data/code references in quiet mode
| axt* [addr]  same as axt, but prints as r2 commands
| axt?         Show this help
[0x00001060]> s main
[0x00001149]> axt?
Usage: axt[?gq*]  find data/code references to this address
| axtj [addr]  find data/code references to this address and print in json format
| axtg [addr]  display commands to generate graphs according to the xrefs
| axtq [addr]  find and list the data/code references in quiet mode
| axt* [addr]  same as axt, but prints as r2 commands
| axt?         Show this help
[0x00001149]> axt
entry0 0x1081 [DATA] lea rdi, [main]
```
Shall I go for it?
Thank you for the guidance, in advance.